### PR TITLE
Update managing-accounts.md

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
- Made sentences precise.
- Corrected grammar.
- Reworded for clarity.

## Description

Describe the changes that were made in this pull request. When possible start with the motivations behind the change. Be sure to also include the following information:

1. Motivation for change?
2. What was changed?
3. Link to relevant issues?

## Checklist

- [ ] [Conventional commits were used](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] New links to files and images were verified
- [ ] For fixes/refactors: all existing references were identified and replaced
- [ ] [Style guide](https://developers.google.com/style) was reviewed and applied
- [ ] Clear code samples were provided
- [ ] People were tagged for review

### CODE REVIEW

**Comment by LakshmiLavanyaKasturi** [comment]
@saralab : Could you review my PR? Thanks.

File: docs/tutorials/managing-accounts.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ Wait a few minutes for the transaction to complete. You can review the status us
 
 :::
 
-Assuming the faucet transaction was successfully processed, you can review the account history. We are expecting at least one transactions to show up in the account history.
+Assuming the faucet transaction was successfull, you can review the account history. We are expecting at least one transaction to show up in the account history.
```
Comment (sabbyanandan) [inline docs/tutorials/managing-accounts.md:163]:
```suggestion
Assuming the faucet transaction was successful, you can review the account history. We are expecting at least one transaction to show up in the account history.
```

**Review by sabbyanandan — CHANGES_REQUESTED** [review]
@LakshmiLavanyaKasturi: Thanks for the contribution! If you could correct the typo, and rebase on top of the latest, that'd be great.

File: docs/tutorials/managing-accounts.md (hunk 1 above)
Comment (LakshmiLavanyaKasturi) [reply docs/tutorials/managing-accounts.md]: Thanks for finding it. Its fixed now.

**Comment by LakshmiLavanyaKasturi** [comment]
@sabbyanandan: This PR is now ready for review.

**Pull request merged**